src/Generator.cpp

```diff
@@ -752,7 +752,7 @@ std::string halide_type_to_c_type(const Type &t) {
 
 int generate_filter_main_inner(int argc, char **argv, std::ostream &error_output) {
     const char kUsage[] =
-        "gengen\n"
+        "gengen \n"
```

Comment:
If you just need something to trigger the buildbots, you can write `git commit --allow-empty -m "trigger buildbots"`... should cut down on churn on this file.

Reply:
No, this is because these tests won't run unless at least one cpp/h fie is touched

Reply:
gah